src/Model/GroundWaterFlow/gwf-lak.f90

```diff
@@ -3091,7 +3091,7 @@ subroutine lak_set_stressperiod(this, itemno)
       bndElem => this%withdrawal(itemno)
       call read_value_or_time_series_adv(text, itemno, jj, bndElem, &
                                          this%packName, 'BND', this%tsManager, &
-                                         this%iprpak, 'WITHDRAWL')
+                                         this%iprpak, 'WITHDRAWAL')
```

Comment:
This typo is possibly the source of a bug (reading the context of the surrounding code)